public/js/components/Scopes.js

```diff
@@ -120,10 +120,18 @@ function getScopes(pauseInfo, selectedFrame) {
         scopes.push({ name: title, path: key, contents: vars });
       }
     } else if (type === "object") {
+      const value = scope.object;
+      // Override preview for Window object
+      if (value.class === "Window") {
+        value.preview = {
+          kind: "Window",
+          isGlobal: true
+        };
+      }
```

Comment:
hmm, interesting - i'm not sure if we have mutated the rep value's before. Curious to see what others think.